modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/storage/DefaultDataStorage.java

```diff
@@ -35,7 +35,10 @@
  * @author Preston Crary
  */
 @Component(
-	property = {"data.storage.type=default", "service.ranking:Integer=100"},
+	property = {
+		"data.storage.type=default", "data.storage.type=json",
+		"service.ranking:Integer=100"
+	},
```

Comment:
@henriquelimadotcom we should not include the `data.storage.type=json` here, we already have the `JSONDataStorage`. 

As far as I know, the `json` type is deprecated, we should only use the `default` from now on.